DOCS.md

```diff
@@ -1054,6 +1054,8 @@ using the '+' character (Example: `file:/home/user/.config/nvim/init.lua +10`) o
 * Headline with `CUSTOM_ID` property within the same file (starts with `#`)
 * Fallback: If file path, opens the file, otherwise, tries to find the headline title.
 
+To open a hyperlink, see |orgmode-org_open_at_point|.
```

Comment:
To link to a specific point in the docs use the markdown link format, like for example here https://github.com/nvim-orgmode/orgmode/blob/7834aea491c02c0eddd2fbc94714535bf06776b3/DOCS.md?plain=1#L814